routes/index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Set up routes for save/publish
+*/
+
+module.exports = function(server) {
+
+  // /projects API route
+  // Handles user-level operations
+  require('./projects/get-projects.js')(server);
+
+  // /project/* API routes
+  // Handles project-level operations
+  require('./project/get-contents.js')(server);
+  require('./project/create.js')(server);
+  require('./project/delete.js')(server);
+  require('./project/publish.js')(server);
```

Comment:
If `delete` actually deletes a project, then you've lost your data when you do `delete` + `create`, where data is any linked content in other tables (e.g., files).  It would force you to re-upload.